tests/test_h1_delete_canonicalization.py

```
@@ -0,0 +1,83 @@
+import pytest
+from hypothesis import example, given, settings
+from hypothesis import strategies as st
+
+from py_hamt import HAMT, InMemoryCAS
+
+Operation = tuple[Literal["set", "delete"], str]
+
+# These numeric strings have the same first byte in their BLAKE3 hashes. The
+# additional keys exercise interleavings outside that colliding root bucket.
+COLLIDING_KEYS = ("5", "15", "123", "317", "349")
+KEY_POOL = COLLIDING_KEYS + ("0", "1", "2", "alpha", "omega")
```

Ruff (0.15.20)

[warning] Consider `(*COLLIDING_KEYS, "0", "1", "2", "alpha", "omega")` instead of concatenation

Replace with `(*COLLIDING_KEYS, "0", "1", "2", "alpha", "omega")`

(RUF005)